.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Stale workflow'
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: '0 0 * * *' # at 00:00 every day
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@main
+        id: stale
+        with:
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs. Thank you
+            for your contributions.
+          days-before-issue-stale: 60
+          days-before-issue-close: 7
```

Comment:
Just double checking, this IS in ADDITION to the `days-before-issue-stale` value right? :) Same for PRs below

Reply:
Yes, 99.9% we have the same today https://github.com/backstage/backstage/blob/master/.github/stale.yml#L4

Reply:
Yeah feels like that should be the same